hack/update-api-reference-docs.sh

```diff
@@ -71,6 +71,7 @@ for ver in $VERSIONS; do
   docker run ${user_flags} \
     --rm -v "${TMP_IN_HOST}":/output:z \
     -v "${SWAGGER_PATH}":/swagger-source:z \
+    --net=host -e "https_proxy=${KUBERNETES_HTTPS_PROXY}" \
```

Comment:
${KUBERNETES_HTTPS_PROXY} is set on the host, see as [proxy-settings](https://github.com/kubernetes/kubernetes/blob/master/build/README.md#proxy-settings)